### DISCUSSION TITLE
Concept score screen

### DISCUSSION THREAD

**Discussion by SIUUUUUUUUUU17** [ppy/osu#30510] [Ideas]
![Untitled35_20241105181314](https://github.com/user-attachments/assets/820b3f9a-0e43-428c-8651-3e2fb223a152)

I just made a concept score screen based on the earlier lazer score screen back in 2018, with changes and features introduced with the current score screen implemented

Would be cool to have something like this

**Comment by Jinkku** [comment]
Idk, I don't really think it'll fit with the argon style.

**Comment by peppy** [comment]
Looks like what we had 3 years ago

**Reply by SIUUUUUUUUUU17** [reply]
Still, I think this looks cooler than the score card that we have in current Lazer builds

No offense, it works better with the Argon styling, but it kind of feels boring to look at for me

**Comment by WitherFlower** [comment]
Having more vision on the background would help identify the beatmap at a glance - one of the stable result screen's strong advantages.
